dhis-2/dhis-test-e2e/src/test/java/org/hisp/dhis/tracker/importer/TrackerExportTests.java

```diff
@@ -456,6 +457,26 @@ public void shouldReturnAscOrderedEventByTEIAttribute() {
         List.of("ZwwuwNp6gVd", "olfXZzSGacW"), events, "Events are not in the correct order");
   }
 
+  @Test
+  public void
+      shouldReturnProgramStageListOrderedByProgramStageDescWhenFieldsAndOrderEqualToProgramStage() {
+    trackerActions
+        .postAndGetJobReport(
+            teiWithEnrollmentAndEventsTemplate, new QueryParamsBuilder().add("async=false"))
+        .validateSuccessfulImport();
+
+    List<String> actualPsList =
+        trackerActions
+            .get("events?order=programStage&fields=programStage&program=f1AyMswryyQ")
+            .validateStatus(200)
+            .extractList("instances.programStage.flatten()");
+
+    List<String> orderedPsList = new ArrayList<>(actualPsList);
+    orderedPsList.sort(Collections.reverseOrder());
```

Comment:
I cannot understand this assertion.
It seems it is asserting that a collection and its reverse should be the same.
I think the expected PsList here could be hardcoded to make it clear what is happening